assets/helpers/page/page.js

```diff
@@ -2,8 +2,16 @@
 
 // ----- Imports ----- //
 
-import { createStore, combineReducers } from 'redux';
 import 'ophan';
+import {
+  createStore,
+  combineReducers,
+  applyMiddleware,
+  compose,
+  type Reducer,
```

Comment:
In order to be consistent to the rest of the codebase, I would keep the type imports in a separate line

Reply:
In all honesty, the rest of the codebase doesn't do this in part because I didn't realise you could until @rupertbates [introduced it](https://github.com/guardian/support-frontend/pull/631#discussion_r184686688). I always assumed the [babel plugin](https://babeljs.io/docs/plugins/transform-flow-strip-types/) couldn't handle stripping them as part of the same import.

I would like to advocate switching to this style, because I think it's cleaner than having two different imports for the same file.

Reply:
After this change, in the same file, we will have both styles. If we would like to change styles, I would update the entire file or keep the re-style for a future PR.

Reply:
2c3914c